src/player_hardcoded_effects.cpp

```diff
@@ -1336,8 +1336,8 @@ void player::hardcoded_effects( effect &it )
                 // Myoclonic seizure (muscle spasm)
                 if( one_turn_in( 2_hours / mod ) && !has_effect( effect_valium ) ) {
                     std::string limb = random_entry<std::vector<std::string>>( { "arm", "hand", "leg" } );
```

Comment:
You also need to add `translate_marker()` around these strings to extract them for translation.

Reply:
Ok added it, is it all good now?